intermediate_html_css/intermediate_html_concepts/introduction.md

```diff
@@ -1,20 +1,20 @@
 ### Introduction
 
-This is the first of the two part course where we dig deeper into HTML and CSS.
+This is the first of the two-part course where we dig deeper into HTML and CSS.
 
-Our foundations lessons by design only scratched the surface of what's possible. We wanted to give you all of the bare necessities for getting started so you could move on and be somewhat productive as quickly as possible, but now it's time to slow down and dig in!
+Our foundation lessons by design only scratched the surface of what's possible. We wanted to give you all of the bare necessities for getting started so you could move on and be somewhat productive as quickly as possible, but now it's time to slow down and dig in!
 
-You have probably realized by now that there are many more HTML elements than we mentioned in our foundations content. Throughout this course, we'll be hitting the rest of the important elements such as forms and tables.
+You have probably realized by now that there are many more HTML elements than we mentioned in our foundation content. Throughout this course, we'll be hitting the rest of the important elements such as forms and tables.
 
-There is also a _lot_ more that you can do with CSS that you'll learn throughout this course, such as variables, functions, shadows and of course grid layouts! So buckle up! By the time you finish this course, you'll be able to recreate just about any web design you can find on the internet... which is an important skill to carry forward. Even if you are not shooting for a front-end specific job, being able to make your portfolio pieces look nice is important when it comes to making yourself stand out.
+There is also a _lot_ more that you can do with CSS that you'll learn throughout this course, such as variables, functions, shadows and of course grid layouts! So buckle up! By the time you finish this course, you'll be able to recreate just about any web design you can find on the internet... which is an important skill to carry forward. Even if you are not shooting for a front-end-specific job, being able to make your portfolio pieces look nice is important when it comes to making yourself stand out.
```

Comment:
I do not believe this needs an hyphen. Having two seems like overkill and I'd rather re-phrase that sentence if you strongly feel it needs to be here.
```
Even if you are not shooting for a front-end job, being able...
```